docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,43 @@
-
-| Library                                                 | Features                   | Notes                                        |
-|---------------------------------------------------------|----------------------------|----------------------------------------------|
-| [Kotlin DataFrame](https://github.com/Kotlin/dataframe) | Structured data processing | Support for `csv`, `json`, and other formats |
-| [Kandy](https://kotlin.github.io/kandy/welcome.html)    | Data visualization         | ...                                          |
-| ...                                                     | ...                        | ...                                          |
+| Library                                                                | Purpose                                                                                           | Features                                                                                                                                                                                                                                                                                                                                                                                                    |
+|------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Tablesaw](https://github.com/jtablesaw/tablesaw)                      | - Data collection <br/> - Data cleaning and processing <br/> - Data exploration and visualization | - Tools for loading, cleaning, transforming, filtering, and summarizing data <br/> -  Inspired by [Plot.ly](https://plotly.com/)                                                                                                                                                                                                                                                                            |
+| [Dex](https://github.com/PatMartin/Dex)                                | - Data collection <br/> - Data cleaning and processing <br/> - Data exploration and visualization | - Tools for reading, transforming, and visualizing data <br/> - Written in Groovy and Java on top of [JavaFX](https://openjfx.io/)                                                                                                                                                                                                                                                                          |
```

Comment:
Note: The last update for this library was 5 years ago

Reply:
Remove dex